src/app/layout.tsx

```diff
@@ -15,9 +16,11 @@ export default function RootLayout({ children }: { children: React.ReactNode })
   return (
     <html lang="en">
       <body className={inter.className}>
-        <DndWrapper>
-          <ThemeRegistry options={{ key: 'mui' }}>{children}</ThemeRegistry>
-        </DndWrapper>
+        <ProviderWrapper>
+          <DndWrapper>
+            <ThemeRegistry options={{ key: 'mui' }}>{children}</ThemeRegistry>
+          </DndWrapper>
```

Comment:
Since all pages do not require Dnd, could we move this out of the root layout file?

Reply:
`<DndWrapper/>` is included here in the root layout because I think we should have draggable functionality in both IU route and Client route. If this is not true, then I will put this only inside the `page.tsx` for main (IU) route. @rrojan

Reply:
@aatbip I was thinking about task detail page mainly... it doesn't depend on Dnd and it could add quite a lot of unused bloat to our app

Reply:
@rrojan This 5a4e21675fe03107c6b5a584b4774eaf5979cfba commit addresses the comment